apps/test/unit/lib/kits/maker/CircuitPlaygroundBoardTest.js

```diff
@@ -125,6 +127,30 @@ describe('CircuitPlaygroundBoard', () => {
         expect(playground.reset).to.have.been.calledOnce;
       });
     });
+
+    it('lets playground-io register its sysex response handler each time', () => {
+      // This test covers a fix for a known accelerometer issue, where the
+      // handler for accelerometer data is from the first Playground object
+      // created on the page.
+      // This is a fragile approach to testing this fix, but reproducing the
+      // real problem in tests is going to be near-impossible since we stub
+      // Firmata at the webpack layer in our tests.
+      expect(Playground.hasRegisteredSysexResponse).to.be.undefined;
```

Comment:
I'm open to suggestions on better ways to harden against this problem, given that we have to stub Firmata.